UI/Web/src/app/manga-reader/manga-reader.component.ts

```diff
@@ -301,10 +300,20 @@ export class MangaReaderComponent implements OnInit, AfterViewInit, OnDestroy {
       !this.isCoverImage(this.pageNum - 1) &&
       !this.isWideImage(this.canvasImage) &&
       !this.isWideImage(this.canvasImageNext) &&
-      window.innerWidth > window.innerHeight
+      window.innerWidth > window.innerHeight // Don't render double reversed if orientation is portrait, mostly mobile
       );
   }
 
+  @HostListener('window:resize', ['$event'])
+  @HostListener('window:orientationchange', ['$event'])
+  onResize() {
+    if (this.layoutMode === LayoutMode.Single || this.readerMode === ReaderMode.Webtoon) return;
+
+    if (window.innerWidth < window.innerHeight) // Maybe do a check before we switch for them
```

Comment:
Remove the if statement since it doesn't do anything. Else scope it correctly by inverting and returning early.

Reply:
Done. First removed it, but then decided to add it again as a guard clause.